build/nsis/Mixxx.nsi

```diff
@@ -546,6 +546,9 @@ Section "Uninstall"
   Delete "$INSTDIR\controllers\Wireless-DJ-scripts.js"
   Delete "$INSTDIR\controllers\Xone K2.midi.xml"
   Delete "$INSTDIR\controllers\Xone-K2-scripts.js"
+  Delete "$INSTDIR\controllers\Numark-Mixtrack-3.midi.xml"
+  Delete "$INSTDIR\controllers\Numark-Mixtrack-3-scripts.js"
```

Comment:
Please keep all files in alphabetical order. This will be easier for future maintenance.
Could you please move these two lines in the correct place ?